src/style.css

```diff
@@ -0,0 +1,98 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format("TrueType");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc") format("TrueType");
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format("TrueType");
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  padding-top: 20px;
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+}
+
+.search-bar {
+  margin-bottom: 20px;
+  padding-left: 62px;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-bar:focus {
+  background: linear-gradient(180deg, #ffff 0%, #f6f6f7 100%);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  font-weight: 900;
+  outline: none;
+}
+
+.search-bar--big {
+  height: 70px;
+  font-size: 16px;
+}
+
+.search-bar--big::placeholder {
+  font-size: 16px;
+}
+
+.search-bar--big:focus {
+  font-size: 16px;
+}
+
+.search-bar--small {
+  padding-left: 33px;
+  height: 42px;
+  font-size: 14px;
+}
```

Comment:
see [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p.2, it can be done easier